picard/tagger.py

```diff
@@ -426,6 +426,7 @@ def exit(self):
         self.webservice.stop()
         self.run_cleanup()
         QtCore.QCoreApplication.processEvents()
+        os._exit(0)
```

Comment:
I'm a bit bothered by this "solution" because it causes the program to exit immediately, without flushing buffers for example.
`exit()` is called from https://github.com/skelly37/picard/blob/5b453543c6ce3a93a84b995ccb9a0148305f67a0/picard/tagger.py#L1025 but we can see we expect some extra code to run after its call.

Also we expect to return an exit code on exit. See https://github.com/skelly37/picard/blob/5b453543c6ce3a93a84b995ccb9a0148305f67a0/picard/tagger.py#L1150 and https://github.com/skelly37/picard/blob/5b453543c6ce3a93a84b995ccb9a0148305f67a0/picard/tagger.py#L443

Reply:
That's true, the only solution I see for now is to replace 

`sys.exit(tagger.run())` with `os._exit(tagger.run())`

So the question is whether it is more important to prevent Picard from freezing or to exit nicely.

Reply:
True, I misread the code. The "dirty" exit should happen as rarely as possible. It's okay now imo.